generate.go

```diff
@@ -43,11 +44,11 @@ func prepareOptions(cmd *cobra.Command, opts *Options) {
 	}
 
 	if opts.Header == "" {
-		opts.Header = strings.Title(opts.Name) + " Manual"
+		opts.Header = cases.Title(language.AmericanEnglish).String(opts.Name) + " Manual"
```

Comment:
I think we need to add language tag to Options, and use english if it's not set. BTW there is also `language.English`, isn't it a better default? Not sure what's the difference.

Reply:
Looking through the docs and guidelines, `language.English` is certainly a better default - thank you for pointing that out!

I will start on these changes and reach out to you again for an additional review once it is completed.

Reply:
I have added a `Language` property to Options that defaults to `en`. If there is an error parsing the user-provided language, it will fall back to `en` silently.